cli/cmd/generate.go

```diff
@@ -75,15 +79,29 @@ func genSource(cmd *cobra.Command, path string, pm *plugin.PluginManager, regist
 		return fmt.Errorf("failed to create source plugin: %w", err)
 	}
 	defer plugin.Close()
-	res, err := plugin.GetClient().ExampleConfig(cmd.Context())
+	client := plugin.GetClient()
+	name, err := client.Name(cmd.Context())
+	if err != nil {
+		return errors.Wrap(err, "failed to get plugin name")
+	}
+	cfg, err := client.ExampleConfig(cmd.Context())
 	if err != nil {
 		return errors.Wrap(err, "failed to get example config")
 	}
-	fmt.Println(res)
+
+	configPath := name + ".yml"
+	if outputFile != "" {
+		configPath = outputFile
+	}
+	err = writeFile(configPath, cfg)
+	if err != nil {
+		return errors.Wrap(err, "failed to write file")
+	}
+	fmt.Fprintln(os.Stderr, "✅ Source plugin config written to "+configPath)
```

Comment:
no emojies please :) otherwise we will have to introduce more flags like no color to stdout and so forth.
also why stderr if it's all prints it successfully ?

Reply:
also let's use `fmt.Errorf` - I was experimenting with errors.Wrap to try and get the line number for sentry but it doesn't work great - I think we should only use it in plugins where it's hard to know where the error came from.

Reply:
stderr because this is a log line and shouldn't be seen as the program's output (unless that is what we want?). Usually log lines go to stderr. At the same time, I didn't want the log line going to the log file, because then the CLI user doesn't see it unless they pass a special flag (that's why I didn't use `log.Info()` here either).

Reply:
I'll take the emoji out 😢

Reply:
I think we should just output to the user (we can also write to the log in addition)

Reply:
stderr is shown to the user, and I think it's the right place for something that is not strictly part of the program's output and is subject to change (like how Go's `log` package also writes to stderr). I'd like to keep it stderr, since I think that's better/more correct, but will change if you think we should still use stdout?